components/input/input-group.component.ts

```diff
@@ -84,17 +84,21 @@ export class NzInputGroupWhitSuffixOrPrefixDirective {
     '[class.ant-input-group-compact]': `nzCompact`,
     '[class.ant-input-search-enter-button]': `nzSearch`,
     '[class.ant-input-search]': `nzSearch`,
+    '[class.ant-input-search-rtl]': `dir === 'rtl'`,
```

Comment:
There are some style issues here. You can see it from the input-group demo. 
![image](https://user-images.githubusercontent.com/50307523/102050096-1f000200-3e1d-11eb-889c-522347dd02bf.png)

Reply:
Done